packages/gatsby-plugin-mdx/utils/gen-mdx.js

```diff
@@ -44,7 +44,7 @@ module.exports = async function genMDX(
 ) {
   const pathPrefixCacheStr = pathPrefix || ``;
   const payloadCacheKey = node =>
-    `gatsby-mdx-entire-payload-${
+    `gatsby-plugin-mdx-entire-payload-${
```

Comment:
@ChristopherBiscardi 	this won't break anything i'm not aware of, right?

Reply:
AFAICT this will just bust any existing caches which shouldn't be an issue.

Reply:
Good call @amberleyromo to ask this question as @johno said, it won't be an issue. 👍